open-x-htb.js

```diff
@@ -128,6 +129,31 @@ function OpenXHtb(configs) {
       auidString = auidString.slice(0, -1);
       ausString = ausString.slice(0, -1);
 
+      // process tdid query param
+      var identityData = returnParcels[0].identityData;
+
+      if (identityData && identityData.AdserverOrgIp && identityData.AdserverOrgIp.data) {
+        // identityData normal format
+        if (identityData.AdserverOrgIp.data.uids) {
+          var adsrvrUids = identityData.AdserverOrgIp.data.uids;
+
+          if (Utilities.isArray(adsrvrUids)) {
+            for (var i = 0; i < adsrvrUids.length; i++) {
+              if (adsrvrUids[i].ext && adsrvrUids[i].ext.rtiPartner === "TDID") {
+                tradeDeskId = adsrvrUids[i].id;
+              }
+            }
+          }
+          // identityData in beta format
```

Comment:
Please advice if support for beta identity data format is needed.
FYI, in https://knowledgebase.indexexchange.com/display/PilotsPro/How+to+Use+adsrvr.org , search for
> (during beta, only the AdserverOrgIp.data.uid.id and AdserverOrgIp.data.uid.ext.rtiPartner objects are available).